irods/test/user_group_test.py

```diff
@@ -606,11 +606,18 @@ def test_set_and_query_user_quota(self):
                 ses.users.remove(bob.name)
 
     def test_we_cannot_overwrite_irodsA_without_overwrite_option_true__issue_635(self):
-        self.assertTrue(os.path.isfile(os.path.expanduser('~/.irods/.irodsA')))
+        # Assert the auth_file exists and store the contents for later reference.
+        auth_file = os.path.expanduser('~/.irods/.irodsA')
+        self.assertTrue(os.path.isfile(auth_file))
+        with open(auth_file,'rb') as f:
+            auth_file_contents = f.read()
+
         with self.assertRaises(client_init.irodsA_already_exists):
             client_init.write_native_credentials_to_secrets_file('somevalue',overwrite=False)
-        with self.assertRaises(client_init.irodsA_already_exists):
-            client_init.write_pam_credentials_to_secrets_file('somevalue',overwrite=False)
```

Comment:
Sorry... not understanding this. Keep which assertions?

Reply:
The diff to which this comment is attached:
```diff
-        with self.assertRaises(client_init.irodsA_already_exists):
-            client_init.write_pam_credentials_to_secrets_file('somevalue',overwrite=False)
```
Was it removed for a reason?

Reply:
I do see that `write_pam_credentials_to_secrets_file` is being tested in one of the BATS test files. Is that why it was removed?

Reply:
Yes, right - that part won't function in the normal run of the test suite because PAM isn't set up, only native, so I'm moving that part of the test (overwrite = False) to become part of test003.

Reply:
I realize now that move wasn't clear while it was happening.

Reply:
No worries. Just wanted to make sure it was still being covered. Resolving this one.